pages/contact-employment-insurance.js

```diff
@@ -198,23 +104,26 @@ export async function getStaticProps({ res, locale }) {
 
   const t = locale === 'en' ? en : fr
 
-  const contactMethods = pareseContactMethods(tmpContactMethods)
+  const pageContent = await getContactEmploymentInsuranceContent().catch(
+    (error) => {
+      logger.error(error)
+      // res.statusCode = 500
+      throw error
+    }
+  )
 
-  const breadCrumbItems =
-    locale === 'en'
-      ? contactMethods.en.breadCrumbs
-      : contactMethods.en.breadCrumbs
+  const breadCrumbItems = locale === 'en' ? [] : []
```

Comment:
const breadCrumbItems = [
    {
      link: t.url_dashboard,
      text: t.pageHeading.title,
    },
  ]
  
  Page is just missing breadCrumbItems like so, this will solve the e2e test error.

Reply:
Added breadcrumbs